### PR TITLE
Add tinygettext

### PR DESCRIPTION
**Pull request by sharkwouter**
This is a translation library with a zlib license that shares a lot with gnu gettext. For some reason it uses uncompiled translation files called .po files. It does work, though.

### CODE REVIEW

File: tinygettext/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+pkgname=tinygettext
+pkgver=20240715
+pkgrel=1
+pkgdesc="a simple gettext replacement that works directly on .po files"
+arch=('mips')
+license=('zlib')
+groups=('pspdev-default')
+url="https://github.com/tinygettext/tinygettext"
+depends=('sdl2')
+optdepends=('')
```
Comment (diamant3) [inline tinygettext/PSPBUILD:10]: Try to remove the empty string from optdepends and see if it builds. It doesn't reflect in the https://pspdev.github.io/psp-packages if it doesn't build all packages properly.

**Review by diamant3 — CHANGES_REQUESTED** [review]

File: tinygettext/PSPBUILD (hunk 1 above)
Comment (sharkwouter) [reply tinygettext/PSPBUILD]: Good catch, I fixed this now

**Comment by diamant3** [comment]
Thanks for adding!

**Comment by bucanero** [comment]
> This is a translation library with a zlib license that shares a lot with gnu gettext. For some reason it uses uncompiled translation files called .po files. It does work, though.

Btw, another ported library for translations is [`mini18n`](https://github.com/pspdev/psp-packages/tree/master/mini18n)
It supports 3 formats: `.po`, `.csv`, and `.yts`

Main difference is licensing, mini18n is GPL2

**Comment by sharkwouter** [comment]
Yeah, I know, I needed something for an MIT licensed project. I think gettext would probably also build.

**Pull request merged**